packages/jsii-rosetta/lib/commands/transliterate.ts

```diff
@@ -99,6 +100,12 @@ export async function transliterateAssembly(
       const now = new Date().getTime();
       // eslint-disable-next-line no-await-in-loop
       const result = await loadAssembly();
+
+      if (result.targets?.[targetName(language)] == null) {
+        // This language is not supported by the assembly, so we skip it...
```

Comment:
Wondering if we need some explicit output to tell callers that a language is unsupported?

Reply:
Yeah but we'd want that ONLY if the language was specifically requested...?